package.json

```diff
@@ -32,11 +32,14 @@
     "@angular/platform-browser-dynamic": "4.0.1",
     "@angular/router": "4.0.1",
     "bulma": "^0.4.0",
+    "electron-simple-updater": "^1.2.0",
     "hterm": "0.0.1",
+    "httpreq": "^0.4.23",
```

Comment:
can you please use [request](https://github.com/request/request) instead of this package?